### ISSUE TITLE
Pause node to adjust and add to the prompts

### ISSUE THREAD

**Issue by serget2** [comfyanonymous/ComfyUI#5965]
### Feature Idea

I am currently running Griptape with Ollama's  and I made a Prompt reverse engineering workflow which works fine mostly. what it does is it takes an image without prompts or generation data, SEES what is in it (So I need Vision LLM's like Lllama 3.2 vision or gemini 1.5) and then tells me what it sees, then I parse it in to another agent that takes that info and turns that in to a prompt, which gives me "Similar" results but off course never perfectly the same, If I wanted that I would just download the pic much easier.
E.g. the original
![12c](https://github.com/user-attachments/assets/e4bb1fc7-e4f2-420b-8abc-bc17e920d106)
vs Mine
![fluxturbo_8_1 0_667](https://github.com/user-attachments/assets/fb101765-6a80-4181-b2c8-b78e624cc647)

BUT sorry for the long intro, LLM's sometimes get it wrong, or have the tendency to block certain things that appear remotely NSFW or change the prompt in such a way that it is SFW, E.g.
the original:
![34AC79C4F7AB09E918773F3B1360D7423644C5DF3E8AA8457E3A1372E98E5ABD](https://github.com/user-attachments/assets/6da64372-74db-4f6b-a1c0-cb29fc73ee46)
vs mine:

![fluxturbo_8_1 0_752](https://github.com/user-attachments/assets/10256037-9722-4a02-80ed-d690c19daca1)

So here is what I need, I want to parse the prompt outside the griptape and LLM sphere in to a node that pauses and asks me to adjust it, or add to it, just a this is the prompt, change it to your needs, then press continue, NOT JUST for the NSFW sometimes it just get's things wrong like instead of making a brutish male it makes an elegant almost female face etc... I need to change things like that before continuing.

TLDR: Make prompt, parse to pause node, adjust, continue

### Existing Solutions

I do not know of any I knew there was some text clip prompt that had a second window which could contain some extra words to add but I do not know which node that was.
Also it did not pause, it just added those words in.

### Other

You do not need to worry about the griptape nodes interfering, The standing spiderwoman has the workflow, it is a basic flux image generator separate from the griptape nodes, so that is not an issue
